src/uwtools/resources/jsonschema/cdeps.jsonschema

```diff
@@ -1,11 +1,92 @@
 {
+  "$defs": {
+    "streams": {
```

Comment:
Since it appears that the `atm_streams` and `ocn_streams` YAML blocks have the same requirements, I factored this block out and then reference it with `$ref` tags in the appropriate sections below.

Reply:
@maddenp-noaa Thanks. Yes, some part of those books are similar but they could also have minor differences.